Dockerfile

```diff
@@ -1,36 +1,57 @@
-# specify the node base image with your desired version node:<version>
-FROM node:12.16.3-slim
+# Build Stage
+FROM node:12.16.3-buster-slim AS umbrel-manager-builder
 
-# Install Tools
+# Install tools
 RUN apt-get update --no-install-recommends \
-    && apt-get install -y --no-install-recommends build-essential g++ \
-    && apt-get install -y --no-install-recommends git \
-    && apt-get install -y --no-install-recommends libltdl7 \
-    && apt-get install -y --no-install-recommends python \
-    && apt-get install -y --no-install-recommends rsync \
-    && apt-get install -y --no-install-recommends vim \
+    && apt-get install -y --no-install-recommends build-essential \
```

Comment:
You don't need to add `--no-install-recommends` since none of this will make it into the final build stage.

You can probably remove a lot of the deps from the Dockerfile if you do that since they'll be included.

Reply:
You're right, but it ends up increasing the build time since a lot of unused dependencies are installed.

Reply:
But overall I prefer it without `--no-install-recommends` too, as it's just simpler and easier to maintain.